docs/usage.md

```diff
@@ -77,7 +101,7 @@ intended version of the Ansible playbooks. A commit hash may be specified by rep
 
 ```screen
```

Comment:
Yeah, the ellipses get highlighted weirdly in that case, IIRC from Jack's original PR for these instructions. Having said that, it might be more helpful to just show an entire CFS configuration here so it's easier to see that this is a layer? I think I'll do that.

Reply:
It was a little more complicated than that because the following paragraph assumes that these two layers were in a single configuration. I ended up combining these two snippets into one YAML snippet.